src/products/mozilla_org/testing.md

```diff
@@ -92,7 +101,7 @@ Please refer to the [guidelines for brands and trademarks](https://mozilla-l10n.
 
 Report issues that can’t be resolved in Pontoon by:
 * Filing a bug in [Bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi?product=Mozilla%20Localizations) and selecting your locale.
-* Adding the l10n PM in charge for NI. The l10n PM information appears at the top left corner on the [project page](https://pontoon.mozilla.org/projects/mozillaorg/).
+* Adding the l10n PM in charge for Need Info. The l10n PM information appears at the top left corner on the project page for [mozilla.org](https://pontoon.mozilla.org/projects/mozillaorg/) and [firefox.com](https://pontoon.mozilla.org/projects/firefoxcom/).
```

Comment:
This should link to the PM page at this point. Also, we should use project manager for consistency with the terminology used in Pontoon.